File: src/main/java/calebzhou/rdi/microservice/model/json/weather/LifeIndex.java

package calebzhou.rdi.microservice.model.json.weather;

import javax.annotation.Generated;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class LifeIndex {

    @SerializedName("ultraviolet")
    @Expose
    private Ultraviolet ultraviolet;
    @SerializedName("comfort")
    @Expose
    private Comfort comfort;

    public Ultraviolet getUltraviolet() {
        return ultraviolet;
    }

    public void setUltraviolet(Ultraviolet ultraviolet) {
        this.ultraviolet = ultraviolet;
    }

    public Comfort getComfort() {
        return comfort;
    }

    public void setComfort(Comfort comfort) {
        this.comfort = comfort;
    }

}

File: src/main/java/calebzhou/rdi/microservice/model/json/weather/LifeIndex.kt
package calebzhou.rdi.microservice.model.json.weather

import com.google.gson.annotations.SerializedName


data class LifeIndex (

  @SerializedName("ultraviolet" ) var ultraviolet : ArrayList<Ultraviolet> = arrayListOf(),
  @SerializedName("carWashing"  ) var carWashing  : ArrayList<CarWashing>  = arrayListOf(),
  @SerializedName("dressing"    ) var dressing    : ArrayList<Dressing>    = arrayListOf(),
  @SerializedName("comfort"     ) var comfort     : ArrayList<Comfort>     = arrayListOf(),
  @SerializedName("coldRisk"    ) var coldRisk    : ArrayList<ColdRisk>    = arrayListOf()

)